xep-0390.xml

```diff
@@ -316,9 +323,9 @@
 000001d9</code>
     <p>Running this octet string through the hash functions leads as to the following &hashset;:</p>
     <code><![CDATA[
-<c xmlns="urn:xmpp:caps" xmlns:hashes="urn:xmpp:hashes:2">
-  <hashes:hash algo="sha-256">kzBZbkqJ3ADrj7v08reD1qcWUwNGHaidNUgD7nHpiw8=</hashes:hash>
-  <hashes:hash algo="sha3-256">79mdYAfU9rEdTOcWDO7UEAt6E56SUzk/g6TnqUeuD9Q=</hashes:hash>
+<c xmlns="urn:xmpp:caps">
+  <hash xmlns="urn:xmpp:hashes:2" algo="sha-256">kzBZbkqJ3ADrj7v08reD1qcWUwNGHaidNUgD7nHpiw8=</hashes:hash>
```

Comment:
The closing tag still uses a prefix, like in the previous revision.

Same for every other case.